introduction-to-qgis.Rmd

```diff
@@ -771,40 +771,40 @@ Create a Print Layout with the content of this project. The map should contain a
 
 Most GIS projects require Georeferencing raster data. Georeferencing is the process of assigning real-world coordinates to each pixel of the raster. These coordinates are often obtained from field surveys - collecting coordinates with a hand-held GPS device for a few easily identifiable features in the image or map. In other cases, where you are looking to digitize the scanned maps with grid markings, the coordinates will be marked on the map image itself. Using these sample coordinates or GCPs (Ground Control Points), the image is warped and made to fit within the chosen coordinate system. 
 
-In this exercise your going to georeference a old scanned map of Banglore, India in 1924. This map is possibly hand-drawn and has no coordinate markings on it so we have to use any other georeferenced map as basemap to locate the features and obtain coordinates. QGIS has a variety of basemaps you can use any of the available map for georeferencing. 
+In this exercise, you're going to georeference an old scanned map of Banglore, India, in 1924. This map is possibly hand-drawn and has no coordinate markings, so we have to use any other georeferenced map as a basemap to locate the features and obtain coordinates. QGIS has a variety of basemaps. You can use any of the available maps for georeferencing. 
```

Comment:
QGIS doesn't have a variety of basemaps. "QGIS has the ability to display basemaps from a variety of sources. We will use the basemap from OpenStreetMap to locate the coordinates of features and georeference the map".